Update/quota mgmt transactions updates (#4639)
getsentry/sentry-docs: src/docs/product/accounts/quotas/index.mdx

Let’s clarify a few terms to start:

> - **Event** - An event is one instance of you sending Sentry data. Generally, this data is an error or a transaction.

Author: ```suggestion
- **Event** - An event is one instance of you sending data to Sentry. Generally, this data is an error or a transaction.
```

> - **Issue** - An issue is a grouping of similar error events, which all share the same fingerprint. Every error event has a set of characteristics, called its fingerprint. For example, Sentry groups events together when they are triggered by the same part of your code. Learn more in our full [Issue Grouping documentation](/product/data-management-settings/event-grouping/).

Author: ```suggestion
- **Issue** - An issue is a grouping of similar error events. Every error event has a set of characteristics called its fingerprint, which is what Sentry uses for grouping. For example, Sentry groups events together when they are triggered by the same part of your code. Learn more in our full [Issue Grouping documentation](/product/data-management-settings/event-grouping/).
```

> - **Attachment** - Attachments are files uploaded in the same request, such as log files. Unless the option to store crash reports is enabled, Sentry will use the files only to create the event, and then drop the files.

Author: ```suggestion
- **Attachment** - Attachments are files uploaded in the same request, such as log files. Unless the option to store crash reports is enabled, Sentry will use these files only to create an event, and then drop them.
```

- **Transaction** - A transaction represents a single instance of a service being called to support an operation you want to measure or track, like a page load. Transaction events are grouped by the transaction name.

> - **Quota** - your quota is the monthly number of events - errors, attachments, and transactions - you pay Sentry to track.

Author: ```suggestion
- **Quota** - Your quota is the monthly number of events — errors, attachments, and transactions — that you pay Sentry to track.
```

3. **Quota availability**

>    Events that exceed your quota are not accepted. Once your event volume is approaching or has exceeded the quota, teammates with the "owner" organization permission level will receive [notification](/product/alerts/notifications/#quota-notifications) emails. To add to your quota or review what happens when you exceed it, see [Increasing Quotas](#increasing-quotas).

Author: ```suggestion
   Events that exceed your quota are not accepted. Once your event volume is approaching or has exceeded the quota, teammates with the "owner" organization permission level will receive [notification](/product/alerts/notifications/#quota-notifications) emails. To learn about adding to your quota or to review what happens when you exceed it, see [Increasing Quotas](#increasing-quotas).
```

4. **Event repetition**

>     Event repeition only applies to error events, not transactions or attachments. The following rules apply for event repeition and your quota:

Author: ```suggestion
    Event repetition only applies to error events, not transactions or attachments. The following rules apply for event repetition and your quota:
```

>     - If you have intervened to "Delete & Discard" an issue, then _future_ error events with the same fingerprint do not count toward your quota.

Author: ```suggestion
    - If you have chosen to "Delete & Discard" an issue, then _future_ error events with the same fingerprint do not count toward your quota.
```

>     - If the issue was previously was resolved, this error event counts toward your quota because it may represent a regression in your code.

Author: ```suggestion
    - If you previously resolved an issue, a new error event counts toward your quota because it may represent a regression in your code.
```

>     - If you have intervened to ignore alerts about error events with the same fingerprint, this event counts toward your quota because the event is still occurring. For more information, see [Inbound Filters](#inbound-data-filters).

Author: ```suggestion
    - If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in [Inbound Filters](#inbound-data-filters).
```

5. **Spike protection**

>    Sentry’s spike protection prevents huge overages in error events from consuming your event capacity. Spike protection is not currently available for transactions. For more information, see [Spike Protection](#spike-protection).

Author: ```suggestion
   Sentry’s spike protection prevents huge overages in error events from consuming your event capacity. Spike protection is not currently available for transactions. Learn more in [Spike Protection](#spike-protection).
```

In addition, depending on your project’s configuration and the plan you subscribe to, Sentry may also check:

6. **Rate limit for the project**

>    If the error event rate limit for the project has been exceeded, and your subscription allows, the event will not be counted. For more information, see [Rate Limiting in our guide to Manage Your Event Stream](/product/accounts/quotas/manage-event-stream-guide/#6-rate-limiting) or [Error Event Limits](/product/accounts/quotas/#error-event-limits). This does not apply to transactions or attachments.

Author: ```suggestion
   If the error event rate limit for a project has been exceeded, and your subscription allows, the event will not be counted. Learn more in [Rate Limiting](/product/accounts/quotas/manage-event-stream-guide/#6-rate-limiting) or in [Error Event Limits](/product/accounts/quotas/#error-event-limits). This does not apply to transactions or attachments.
```

7. **Inbound filters**

>    If any inbound filter is set for this type of error event, and your subscription allows, the event will not be counted. For more information, see [Inbound Filters in our guide to Manage Your Event Stream](/product/accounts/quotas/manage-event-stream-guide/#3-inbound-data-filters) or [Inbound Data Filter](#inbound-data-filters). This does not apply to transactions or attachments.

Author: ```suggestion
   If any inbound filter is set for a type of error event, and your subscription allows, the event will not be counted. Learn more in the [Inbound Filters section of our error quota management guide](/product/accounts/quotas/manage-event-stream-guide/#3-inbound-data-filters) or in [Inbound Data Filter](#inbound-data-filters). This does not apply to transactions or attachments.
```
